### PR TITLE
Update ricky hero svgs to use the ones on Notion, adjust color of site

### PR DESCRIPTION
**Pull request by sjv01180**

### CODE REVIEW

**Review by peterkos — CHANGES_REQUESTED** [review]
Changes look good overall, but pls make sure to update your `node` version is 15 or greater! 

Delete the `package-lock.json`, update node, and re-run `npm install`; there should be no changes to the `package-lock.json` after

**Review by sjv01180 — COMMENTED** [review]
Changed package-lock with a more recent nodejs version. Shoutout to my coop for insisting on limiting nodejs to 6.8.0

**Comment by peterkos** [comment]
@sjv01180 Can you also follow the steps above of deleting package-lock, `npm install`, then re-commit?

File: index.js [hunk 1]
```diff
@@ -142,4 +142,4 @@ $('.day-second').click(function() {
     $('.day-first').removeClass('day-active');
     $('.day-second-events').show();
     $('.day-second').addClass('day-active');
-});
+});
```
Comment (peterkos) [inline index.js:145]:
```suggestion
});

```
nit: newline end of file

**Review by peterkos — CHANGES_REQUESTED** [review]
one nit!

**Review by peterkos — APPROVED** [review]
🎉

**Pull request merged**